update_remote_settings_records.mjs

```diff
@@ -5,11 +5,10 @@
 /* global process */
 
 // This script consumes the following env variables:
-// - FX_REMOTE_SETTINGS_WRITER_USER (mandatory): User
-// - FX_REMOTE_SETTINGS_WRITER_PASS (mandatory): Password
-// - FX_REMOTE_SETTINGS_WRITER_SERVER (mandatory): Writer server URL (eg. https://settings-writer.stage.mozaws.net/v1)
-// - FX_REMOTE_SETTINGS_ENVIRONMENT (optional): dev, stage, prod. When set to `dev`, the script will approve its own changes.
-// - FX_REMOTE_SETTINGS_DRY_RUN (optional): If set to 1, no changes will be made to the collection, this will
+// - AUTHORIZATION (mandatory): Raw authorization header (e.g. `AUTHORIZATION='Bearer XXXXXXXXXXXXX'`)
+// - SERVER (mandatory): Writer server URL (eg. https://settings-writer.stage.mozaws.net/v1)
+// - ENVIRONMENT (optional): dev, stage, prod. When set to `dev`, the script will approve its own changes.
+// - DRY_RUN (optional): If set to 1, no changes will be made to the collection, this will
```

Comment:
Is there a reason not to keep the `FX_REMOTE_` prefix for the env variables?

Reply:
yes, @leplatrem asked us to follow https://remote-settings.readthedocs.io/en/latest/support.html#how-do-i-automate-the-publication-of-records-forever, sorry for the confusion, I was told to follow https://github.com/mozilla/passwordmgr-remote-settings-updater's example before ..